server/events_controller.go

```diff
@@ -435,6 +439,20 @@ func (e *EventsController) handleCommentEvent(w http.ResponseWriter, baseRepo mo
 		return
 	}
 
+	// Check if the user who commented has the permissions to execute the 'plan' or 'apply' commands
+	if !e.TestingMode {
+		ok, err := e.checkUserPermissions(baseRepo, user, parseResult.Command)
```

Comment:
I think moving this into `CommandRunnner` might work better. If there is ever another way to run atlantis outside of  GitHub PRs it will allow developers not to add another check.